src/Microsoft.Diagnostics.Monitoring.RestServer/StorageOptions.cs

```diff
@@ -15,6 +16,7 @@ namespace Microsoft.Diagnostics.Monitoring.RestServer
 {
     internal class StorageOptions
     {
+        [Display(Description = "The location for temporary artifacts, such as dump files. Defaults to the temp folder.")]
```

Comment:
I know I'm being pedantic, but since it is named `DumpTempFolder`, it should be the temp folder ONLY for dump files.